### DISCUSSION TITLE
Release v1.6.3 / 5.61.3

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2554] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This is a maintenance release fixing various bugs.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

# Changelog

### Changed
- improved Network Location Awareness template

### Fixed
- fixed issue with update label when "version update" is set to "ignore"
- fixed issue with wow64 thunking on 32 bit windows 10 systems [#2546](https://github.com/sandboxie-plus/Sandboxie/issues/2546)
- fixed on reboot auto clean up boxes are nto cleared [#2531](https://github.com/sandboxie-plus/Sandboxie/issues/2531)
-- Note: a clean up is now done after each reboot
- fixed issue with Network Location Awareness under windows 11 [#2530](https://github.com/sandboxie-plus/Sandboxie/issues/2530)
- fixed issues with recovery window [#2458](https://github.com/sandboxie-plus/Sandboxie/issues/2458)

### Fixed
- Fixed crash issue in Vintage UI introduced in 1.6.1a

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.6.3'>Release v1.6.3 / 5.61.3</a>.</em>

**Comment by marcinkit** [comment]
tested 5.61.3 on win10 x32 using chromium browser, all good here no errors, tyvm.

**Comment by marcinkit** [comment]
i made a monetary donation of 5 eur thru paypal.
